packages/rate-limit/rate-limit.js

```diff
@@ -133,7 +143,10 @@ RateLimiter.prototype.check = function (input) {
   var reply = {
     allowed: true,
     timeToReset: 0,
-    numInvocationsLeft: Infinity
+    numInvocationsLeft: Infinity,
+    userId: input.userId,
+    clientAddress: input.clientAddress,
+    connectionId: input.connectionId,
```

Comment:
Looking at https://github.com/meteor/meteor/issues/5541#issuecomment-272126029 it seems having access to the `name` is what @sebakerckhof was looking for.  I'm not sure it makes sense to expand all of the properties into the reply.  As an alternative, do you have any opinion on my suggestion above (in regards to passing `input` as a separate argument to the callback OR providing the `input` object in the reply as `reply.input`)?